### PR TITLE
First translation

### PR DESCRIPTION
**Pull request by igor-dyatlov**

### CODE REVIEW

File: po/ru.po [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 "
+"|| n%100>=20) ? 1 : 2);\n"
+"X-Generator: Poedit 3.1\n"
+
+#: data/app.drey.EarTag.desktop.in:3 data/app.drey.EarTag.appdata.xml.in:3
+msgid "Ear Tag"
+msgstr "Ушная бирка"
```
Comment (knuxify) [inline po/ru.po:25]: II don't think that the name should be translated; despite being named after a real-life thing I treat it more like an original name. I'll probably update the files to include a note for translators about this.

(I should probably add a "generic name" field in the .desktop file as a more translation-friendly alternative; for now the description sort of covers that.)

**Review by knuxify — CHANGES_REQUESTED** [review]
LGTM overall, thanks for the contribution! I've just got one note about translating the app name (see review), that's mostly my fault for not making it clear earlier :p

File: po/ru.po (hunk 1 above)
Comment (igor-dyatlov) [reply po/ru.po]: I returned original name and added a comment it to the template for translators

**Comment by igor-dyatlov** [comment]
Maybe there is a better way to use something like `translatable="false"`

**Pull request merged**